frontend/src/router.js

```diff
@@ -164,6 +188,22 @@ function requireAuth (to, from, next) {
   })
 }
 
+async function requireCamp (to, from, next) {
+  await campFromRoute(to).call({ api: { get } })._meta.load.then(() => {
+    next()
+  }).catch(() => {
+    next({ name: 'home' })
```

Comment:
Ich glaubs ich fänds schöner dem Benutzer eine Fehlermeldung anzuzeigen anstatt einfach auf home umzuleiten.

Reply:
Aber ums Umleiten auf eine "safe" URL kommen wir nicht herum, sonst läuft das Reaktivitäts-System von Vue Amok indem es immer wieder versucht das Camp zu laden. Wir können ein eigenes Issue aufmachen für ein globales Alert-Banner das ich via Vuex befüllen kann, aber in diesem PR fixe ich erst mal die endlosen 404er Netzwerk-Requests. Wärst du damit einverstanden?

Reply:
D'accord